portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase

```diff
@@ -28,6 +28,84 @@ definition {
 		}
 	}
 
+	@description = "LPS-100416 - Non-Repeatable text fields show incorrect translation on label at first load."
+	@priority = "4"
+	test AddNonRepeatableTextFieldWithTranslation {
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+		NavItem.gotoStructures();
+
+		WebContentStructures.addCP(structureName = "WC Structure Name");
+
+		DataEngine.addField(
+			fieldFieldLabel = "English",
+			fieldName = "Text");
+
+		Click(
+			key_fieldFieldLabel = "English",
+			locator1 = "DDMEditStructure#FORM_FIELD_CONTAINER_LABEL");
+
+		Navigator.gotoNavTab(navTab = "Advanced");
+
+		Uncheck.uncheckToggleSwitch(
+			key_fieldName = "repeatable",
+			locator1 = "Checkbox#SWITCH");
```

Comment:
@Jayda-4145 there might be an existing macro for these steps - let's try to make sure individual functions are placed in macros, then we can use them later on.  This time it's ok.

Reply:
@brianwulbern ok, got it! Thanks. I will try next time.